jcl/src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -278,12 +278,17 @@ static final void initializeClassLoaders() {
 		/*[ENDIF]*/
 		jdk.internal.misc.VM.initLevel(2);
 		String javaSecurityManager = System.internalGetProperties().getProperty("java.security.manager"); //$NON-NLS-1$
-		if (null != javaSecurityManager) {
-			if (javaSecurityManager.isEmpty() || "default".equals(javaSecurityManager)) {
+		if ((javaSecurityManager != null) 
+		/*[IF Java12]*/
+			&& !javaSecurityManager.equals("disallow") //$NON-NLS-1$ /* special token to disallow SecurityManager */
```

Comment:
Please add a comment to refer to the SecurityManager javadoc.

Reply:
`JDK 12` doc is still an [early access](https://download.java.net/java/early_access/jdk12/docs/api/java.base/java/lang/SecurityManager.html), added a comment `More details are at JDK 12 javadoc jdk12/docs/api/java.base/java/lang/SecurityManager.html` instead.

Please have another look.

Reply:
I didn't mean to include a direct link. In fact, I don't think the path is desired either. I'm fine with a comment at says "See the SecurityManager javadoc for details."

Reply:
Sure, the comment is updated.